src/main/webapp/view/discussion/discussion-comment.js

```diff
@@ -64,25 +64,52 @@ export default class DiscussionComment extends HTMLElement {
    */
   setComment(comment) {
     this.comment = comment;
-    this.setSlotSpan(
-        DiscussionComment.#SLOT_HEADER, this.getHeaderString(comment));
-    this.setSlotSpan(DiscussionComment.#SLOT_CONTENT, comment.content);
+    this.setHeader();
+    this.setContent();
     this.setTypeTag(comment.type);
   }
 
+  /**
+   * Adds the comment's header to the header slot. For root comments, adds
+   * a click event handler to seek everything to the comment's timestamp.
+   */
+  setHeader() {
+    const headerSpan = document.createElement('span');
```

Comment:
`'span'` is used multiple times, consider making it a class const

Reply:
I think that built-in strings for CSS/events/DOM stuff should remain as literals